### PR TITLE
GROW-358 Use a contentful image for the appeal banner

### PR DESCRIPTION
**Pull request by ryan-ludwig**
image

### CODE REVIEW

File: src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBannerCircular.vue [hunk 1]
```diff
@@ -159,7 +148,7 @@ export default {
 			type: String,
 			default: '',
 		},
-		image: {
+		imageUrl: {
 			type: String,
 			default: ''
```
Comment (mcstover) [inline src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBannerCircular.vue:153]: I think we need a default here that's not controlled in contentful...

**Review by mcstover — APPROVED** [review]
Looks good.

We should probably consider adding an id field to the Responsive Image Content Type that is more code friendly...but in the meantime just adding documentation for how this should be used is prob fine.

**Comment by eddieferrer** [comment]
Looks good. I think we're getting a little hacky with the Responsive Image Set on contentful, and should maybe consider adding a different content type that just links to a single item of media for cases like this...

**Review by eddieferrer — APPROVED** [review]

**Comment by mcstover** [comment]
> Looks good. I think we're getting a little hacky with the Responsive Image Set on contentful, and should maybe consider adding a different content type that just links to a single item of media for cases like this...

We could make the Media type available so we can include the media directly when it doesn't require any responsive versions.

**Comment by ryan-ludwig** [comment]
I agree, 

> Looks good. I think we're getting a little hacky with the Responsive Image Set on contentful, and should maybe consider adding a different content type that just links to a single item of media for cases like this...

**Comment by ryan-ludwig** [comment]
To follow up, we'll need to come back around to changing the media type in the future.

**Pull request merged**